README.md

```diff
@@ -65,51 +69,59 @@ clojure -Tclj-watson scan :p deps.edn
-widely used method among the Clojure/Java SCA tools. It downloads all
-vulnerabilities from NVD and stores it in a database (located in the `/tmp/db/`
-folder), composes a
-[Common Platform Enumeration (CPE)](https://nvd.nist.gov/products/cpe) based on
-the dependencies, scans all JARs in the classpath and matches vulnerabilities
-using it.
+widely used method among the Clojure/Java SCA tools. It:
+1. Downloads a database of known vulnerabilities from [NIST NVD](https://nvd.nist.gov/), storing it locally under your `/tmp/db/` folder
+3. Scans JARs from dependencies specified in your `deps.edn`
+4. Composes a [Common Platform Enumeration (CPE)](https://nvd.nist.gov/products/cpe) based on your dependencies
+5. Returns any matching vulnerabilities
 
-* `clj-watson` v5.1.3 onward uses DependencyCheck 10.0.x and the new NIST NVD API.
-  * `clj-watson` v5.0.0..v5.1.2 used DependencyCheck 9.0.x which caused the NIST NVD API to be overwhelmed; please update to v5.1.3!
-* `clj-watson` v4.x.x uses an earlier version of DependencyCheck and the old NVD data feeds, which have been deprecated.
+`clj-watson` then takes reports these findings to you, optionally with [potential remediations](#remediation-suggestions).
 
-#### NIST NVD API
+> [!IMPORTANT]
+> We _always_ recommend using the latest clj-watson, but as a minimum upgrade to v5.1.3.
+> All earlier versions of clj-watson are officially deprecated.
+> Older versions of clj-watson use older problematic versions of DependencyCheck, which NIST is now blocking.
 
-As of version v5.0.0, `clj-watson` switched to
-[`DependencyCheck` 9.0.x](https://github.com/jeremylong/DependencyCheck/tree/main?tab=readme-ov-file#900-upgrade-notice)
-which switches from the earlier NVD data feeds to the new NIST NVD API.
-**NIST are forcing everyone to upgrade to 10.0.3 or later so please use `clj-watson` v5.1.3 or later!**
+#### NIST NVD API
 
-This new API heavily throttles anonymous requests, so it is
-[highly recommended to get an API key](https://github.com/jeremylong/DependencyCheck/tree/main?tab=readme-ov-file#nvd-api-key-highly-recommended)
-in order to use the API efficiently.
+> [!IMPORTANT]
+> The [NIST NVD data feeds discourage access without API keys by heavily throttling anonymous requests](https://nvd.nist.gov/general/news/API-Key-Announcement).
+> So, request one and use it.
 
-Read the [NIST NVD announcement](https://nvd.nist.gov/general/news/API-Key-Announcement) for more information.
+It is easy to [request an API key](https://github.com/jeremylong/DependencyCheck/tree/main?tab=readme-ov-file#nvd-api-key-highly-recommended).
 
-Once you have an API key, you can provide it to `clj-watson` via the `nvd.api.key`
-property in the optional `clj-watson.properties` file, either on the classpath
-you use to run `clj-watson` or via the `-w` / `--clj-watson-properties`
-command-line option:
+You'll specify your key in your `clj-watson.properties` file:
```

Comment:
Or via the `nvd.api.key` JVM property, right? That may be an easier option for most users, so maybe mention that first and show an example with `-J-Dnvd.api.key=...` or perhaps via `:jvm-opts` in the `:clj-watson` alias?

Reply:
I was waiting to see what comes out of #66. 
But I think the JVM property will always be an option, because DependyCheck looks for it, so ya, good idea, I'll describe it now.

Reply:
I described setting the key via the system property. Lemme know what you think.